modules/nf-core/freebayes/tests/main.nf.test.snap

```diff
@@ -0,0 +1,14 @@
+{
+    "test.vcf.gz": {
+        "content": [
+            "test.vcf.gz"
+        ],
+        "timestamp": "2023-12-12T13:33:01.830845"
+    },
+    "versions.yml": {
+        "content": [
+            "versions.yml"
```

Comment:
@mashehu can we lint for presence of md5 for versions in the snap ?

Reply:
Yes, please write an issue